lib/mock_suey/mock_contract.rb

```diff
@@ -19,7 +19,7 @@ def initialize(contract, msg)
       def captured_calls_message(calls)
         calls.map do |call|
           contract.args_pattern.map.with_index do |arg, i|
-            (ANYTHING == arg) ? "_" : call.arguments[i].inspect
+            (arg == ANYTHING) ? "_" : call.arguments[i].inspect
```

Comment:
Just out of curiosity: why you swapped these two arguments of `==` operator?

Reply:
I reversed these arguments because rubocop showed me a warning `Style/YodaCondition`. [link to the rule](https://docs.rubocop.org/rubocop/cops_style.html#styleyodacondition).

It looks a bit more logical to put the `arg` first: `"argument equals constant" `vs `"constant equals argument"`.
Anyway, I don't mind both kinds of comparations. It is easy to read both of them.